File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/03.Conditional-Statements-Advanced/03-additional/04.carToGo.js
function carToGo(input) {

    const budget = Number(input[0]);
    const season = input[1];

    let price = 0;
    let type = 0;
    let carClass = 0;

    if (budget <= 100) {
        carClass = "Economy class";
        switch (season) {
            case "Summer": type = "Cabrio";
                price = 0.35 * budget;
                break;
            case "Winter": type = "Jeep";
                price = 0.65 * budget;
                break;
        }
    } else if (budget <= 500) {
        carClass = "Compact class";
        switch (season) {
            case "Summer": type = "Cabrio";
                price = 0.45 * budget;
                break;
            case "Winter": type = "Jeep";
                price = 0.80 * budget;
                break;
        }
    } else if (budget > 500) {
        carClass = "Luxury class";
        type = "Jeep";
        price = 0.9 * budget;
    }

    console.log(`${carClass}`);
    console.log(`${type} - ${price.toFixed(2)}`);
}
carToGo(["99.99", "Summer"]);

File: Python_Basics/Homeworks-And-Labs/03.conditional_statements_advanced/03.additional/04.car_to_go.py
budget = float(input())
season = input()

price = 0
type = 0
carClass = 0

if budget <= 100:
    carClass = "Economy class"
    if season == 'Summer':
        type = "Cabrio"
        price = 0.35 * budget
    elif season == 'Winter':
        type = "Jeep"
        price = 0.65 * budget
elif budget <= 500:
    carClass = "Compact class"
    if season == 'Summer':
        type = "Cabrio"
        price = 0.45 * budget
    elif season == 'Winter':
        type = "Jeep"
        price = 0.80 * budget
elif budget > 500:
    carClass = "Luxury class"
    type = "Jeep"
    price = 0.9 * budget

print(carClass)
print(f'{type} - {price:.2f}')
